AdminWebsite/AdminWebsite.UnitTests/Mappers/ParticipantResponseMapperTest.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using AdminWebsite.Contracts.Enums;
+using AdminWebsite.Contracts.Responses;
+using AdminWebsite.Mappers;
+using BookingsApi.Contract.V1.Responses;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace AdminWebsite.UnitTests.Mappers
+{
+    public class ParticipantResponseMapperTest
+    {
+        [Test]
+        public void Should_map_person_response_to_participant_response_V1()
+        {
+            var id = Guid.NewGuid();
+            List<BookingsApi.Contract.V1.Responses.ParticipantResponse> participants = new List<BookingsApi.Contract.V1.Responses.ParticipantResponse>();
+            BookingsApi.Contract.V1.Responses.ParticipantResponse participant = new BookingsApi.Contract.V1.Responses.ParticipantResponse()
+            {
+                FirstName = "Sam",
+                LastName = "Smith",
+                ContactEmail = "judge@personal.com",
+                HearingRoleName = "Judge",
+                DisplayName = "Display Name",
+                Id = id
+            };
+            participants.Add(participant);
+
+            var participantsResponse = ParticipantResponseMapper.Map(participants);
+
+            foreach (var participantResponse in participantsResponse)
```

Comment:
could we assert on all of the mapped properties here?

Id
DisplayName
CaseRoleName
HearingRoleName
UserRoleName
Title
FirstName
MiddleNames
LastName
ContactEmail
TelephoneNumber
Username
Organisation
Representee
LinkedParticipants

Reply:
done